modules/ROOT/pages/_partials/partner-and-customer-guide.adoc

```diff
@@ -45,12 +45,11 @@ To decouple the minimum Mule version when running tests, add the `-Dmunit.jvm` p
 Before upgrading and testing your custom connector for Java 17, you must be familiar with the following considerations:
 
 * Target compilation level
+From Mule 4.9.0 and onwards, connectors can be compiled with any Java versions providing it can generate bytecode the proper version.
 +
-NOTE: The maximum supported version for running your applications is Java 17 for Mule runtime 4.6.x and later. During the packaging of your app, all code, including third-party dependencies, *must* be compiled for Java 8.
+For this, the version of `mule-modules-parent` must be upgraded to 1.9.0 of higher, and the `java.release.version` set to the minimal Java version that the connector will require. (See xref:java-version-support.adoc#bytecode_level[Bytecode Level Configuration]).
```

Comment:
> must be upgraded to 1.9.0 of higher, and the `java.release.version` set to the minimal Java version that the connector will require

This is incorrect. Setting the parent pom to 1.9.0 will make the connector's MMV to also be 4.9.0. Therefore, there's no point in lowering the language level as there's no way to run that connector in any other mule version. The parent pom should already be pointing to Java 17 and we shouldn't tell customers how to change that

This documentation should make it clear that using 1.9.0 uplifts the MMV and that the connector will Java java 17 as the only supported Java version (e.g: using `@JavaVersionSupport({JAVA_8, JAVA_11, JAVA_17})` wouldn't make sense, they will have to use `@JavaVersionSupport(JAVA_17)`